hoot-core/src/main/cpp/hoot/core/visitors/geometrymodifiers/GeometryModifierAction.h

```diff
@@ -37,8 +37,6 @@
 // boost
 #include <boost/shared_ptr.hpp>
 
-using namespace boost;
```

Comment:
`using namespace` in a header file can cause a lot of unintended problems.  Reserve that for source files if needed.